pkgs/applications/video/aegisub/default.nix

```
@@ -2,162 +2,214 @@
 , config
 , stdenv
 , fetchFromGitHub
-, boost179
+, fetchurl
+, meson
+, ninja
 , cmake
-, expat
-, harfbuzz
-, ffmpeg
-, ffms
-, fftw
+, pkg-config
+, luajit
+, gettext
+, python3
+, wrapGAppsHook
 , fontconfig
-, freetype
-, fribidi
-, glib
-, icu
-, intltool
-, libGL
-, libGLU
-, libX11
 , libass
-, libiconv
+, boost
+, wxGTK32
+, icu
+, ffms
+, fftw
 , libuchardet
-, luajit
-, pcre
-, pkg-config
-, which
-, wrapGAppsHook
-, wxGTK
+, ffmpeg_4
+, jansson
+, libGL
 , zlib
-
+, libX11
 , spellcheckSupport ? true
 , hunspell ? null
-
 , openalSupport ? false
 , openal ? null
-
 , alsaSupport ? stdenv.isLinux
 , alsa-lib ? null
-
 , pulseaudioSupport ? config.pulseaudio or stdenv.isLinux
 , libpulseaudio ? null
-
 , portaudioSupport ? false
 , portaudio ? null
-
-, useBundledLuaJIT ? false
-, darwin
+,
 }:
-
 assert spellcheckSupport -> (hunspell != null);
 assert openalSupport -> (openal != null);
 assert alsaSupport -> (alsa-lib != null);
 assert pulseaudioSupport -> (libpulseaudio != null);
-assert portaudioSupport -> (portaudio != null);
+assert portaudioSupport -> (portaudio != null); let
+  inherit (lib) optional;
 
-let
   luajit52 = luajit.override { enable52Compat = true; };
-  inherit (lib) optional;
-  inherit (darwin.apple_sdk.frameworks) CoreText CoreFoundation AppKit Carbon IOKit Cocoa;
+
+  full_version = "feature_11";
+
+  bestsource = fetchFromGitHub {
+    owner = "vapoursynth";
+    repo = "bestsource";
+    rev = "ba1249c1f5443be6d0ec2be32490af5bbc96bf99";
```

Please provide links to find these revisions, for example

```suggestion
    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/bestsource.wrap#L3
    rev = "ba1249c1f5443be6d0ec2be32490af5bbc96bf99";
```